src/System.IO.FileSystem/tests/FileSystemTest.cs

```diff
@@ -35,6 +35,13 @@ public string GetTestFilePath([CallerMemberName]string fileName = null, [CallerL
             return Path.Combine(TestDirectory, String.Format("{0}_{1}", fileName ?? "testFile", lineNumber));
         }
 
+        // Generates a test file path to use that is unique name per test case / call with an additional
+        // variable to specify a trailing identifier
+        public string GetTestFilePath(int index, [CallerMemberName]string fileName = null, [CallerLineNumber] int lineNumber = 0)
```

Comment:
Yes, it should be memberName; I'll include the name change.

Agreed. The FileSystemTest class isn't specific to FileSystem, despite the name.